standard/clause_0_front_material.adoc

```diff
@@ -43,3 +43,4 @@ All questions regarding this submission should be directed to the editor or the
 == Acknowledgements
 
 This work was inspired by a demonstration of the concept by Joan Masó of CREAF.
+2
```

Comment:
?

Reply:
@letmaik There was an acknowlegement of Joan Maso in the original spec. What words, that give him appropriate awckowlegment, would you prefer?

Reply:
Removed.